Description: Include libavcodec/vdpau.h as C library
 This fixes a linking failure with av_vdpau_alloc_context.

Author: Andreas Cadhalpun <Andreas.Cadhalpun@googlemail.com>
Last-Update: <2014-10-24>

--- xbmc-13.2+dfsg1.orig/xbmc/cores/dvdplayer/DVDCodecs/Video/VDPAU.h
+++ xbmc-13.2+dfsg1/xbmc/cores/dvdplayer/DVDCodecs/Video/VDPAU.h
@@ -43,7 +43,6 @@
 #include "DllAvUtil.h"
 #include "DVDVideoCodec.h"
 #include "DVDVideoCodecFFmpeg.h"
-#include "libavcodec/vdpau.h"
 #include <X11/Xlib.h>
 #include <X11/Xutil.h>
 #define GLX_GLXEXT_PROTOTYPES
@@ -52,7 +51,6 @@
 #include "DllAvUtil.h"
 #include "DVDVideoCodec.h"
 #include "DVDVideoCodecFFmpeg.h"
-#include "libavcodec/vdpau.h"
 #include "threads/CriticalSection.h"
 #include "threads/SharedSection.h"
 #include "settings/VideoSettings.h"
@@ -62,6 +60,14 @@
 #include "utils/ActorProtocol.h"
 #include <list>
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+#include "libavcodec/vdpau.h"
+#ifdef __cplusplus
+}
+#endif
+
 using namespace Actor;
 
 
